code/obj/machinery/singularity.dm

```diff
@@ -76,35 +80,40 @@ Contains:
 	event_handler_flags = IMMUNE_SINGULARITY
 	deconstruct_flags = DECON_WELDER | DECON_MULTITOOL
 
-	bound_width = 96
-	bound_height = 96
-	bound_x = -32
-	bound_y = -32
+	bound_width = 32
+	bound_height = 32
+	bound_x = 16
+	bound_y = 16
```

Comment:
you might want these to scale with size....

Reply:
scaling them created a weird bug that only manifested when the singularity got loose, not having it scale seems to not create any bugs. going to leave it as is until it breaks

Reply:
Setting these vars to something else than multiples of 32 makes the game very choppy due to byond internal stuff.

Hoenstly if you are scaling the singulo boundary down to one tile I'd just remove these 4 lines
```suggestion
```
and then update the `get_center()` proc to just return `loc`. As you can see by my earlier exasperated comment in the `get_center()` proc it's all sorta messy. Then you could also remove the ugly `has_moved` var.